docs/source/example_notebooks/gcm_401k_analysis.ipynb

```diff
@@ -0,0 +1,335 @@
+    "|db|Covariate|defined benefit pension|\n",
+    "|marr|Covariate|married?|\n",
+    "|twoearn|Covariate|two earners|\n",
+    "|pira|Covariate|participation in IRA|\n",
+    "|hown|Covariate|home owner?|\n",
+    "|hval|Covariate|home value (in USD)|\n",
+    "|hequity|Covariate|home equity (in USD)|\n",
+    "|hmort|Covariate|home mortgage (in USD)|\n",
+    "|nohs|Covariate|no high-school? (one-hot encoded)|\n",
+    "|hs|Covariate|high-school? (one-hot encoded)|\n",
+    "|smcol|Covariate|some-college? (one-hot encoded)|\n",
+    "\n",
+    "\n",
+    "The dataset is publicly available online from the [`hdm`](https://rdrr.io/cran/hdm/man/pension.html) R package. For more details about the data set, we refer the interested reader to the following paper:\n",
+    "\n",
+    "V. Chernohukov, C. Hansen (2004). [The impact of 401(k) participation on the wealth distribution: An instrumental quantile regression analysis](http://www.mit.edu/~vchern/papers/ch_401k.pdf). The Review of Economic and Statistics 86 (3), 735–751. \n",
+    "\n",
+    "Let's load and analyse the data first."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "df = pd.read_csv(\"pension.csv\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Effect of 401(k) Eligibility on Net Financial Assets, Conditioned on Income\n",
+    "\n",
+    "First we construct a causal graph of 401(k) plan eligibility (the treatment $T$), net financial assets (the outcome $Y$), control variables $W$ we adjust for assuming that they block all back-door paths between $Y$ and $T$, and income $X$ (the covariate of interest based on which we want to study the heterogeneity of treatment effect)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import networkx as nx\n",
+    "import dowhy.gcm as gcm\n",
+    "\n",
+    "treatment_var = \"e401\"\n",
+    "outcome_var = \"net_tfa\"\n",
+    "covariates = [\"age\",\"inc\",\"fsize\",\"educ\",\"male\",\"db\",\n",
+    "              \"marr\",\"twoearn\",\"pira\",\"hown\",\"hval\",\n",
+    "              \"hequity\",\"hmort\",\"nohs\",\"hs\",\"smcol\"]\n",
+    "\n",
+    "edges = [(treatment_var, outcome_var)]\n",
+    "edges.extend([(covariate, treatment_var) for covariate in covariates])\n",
+    "edges.extend([(covariate, outcome_var) for covariate in covariates])\n",
+    "\n",
+    "causal_graph = nx.DiGraph(edges)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "plt.rcParams['figure.figsize'] = [20, 20] # Make plots larger\n",
```

Comment:
Wondering if we should introduce an optional parameter to the plot function to make this easier. (Separate PR)

Reply:
Yea would make sense. Although this is a global parameter (might need to restore original values after plotting) and it is typically only an issue in the jupyter notebooks. But we can add it as an optional parameter anyways.